whipper/command/mblookup.py

```diff
@@ -1,46 +1,57 @@
 from whipper.command.basecommand import BaseCommand
-from whipper.common.mbngs import musicbrainz
+from whipper.common.mbngs import musicbrainz, getReleaseMetadata
 
 
 class MBLookup(BaseCommand):
     summary = "lookup MusicBrainz entry"
-    description = """Look up a MusicBrainz disc id and output information.
+    description = """
+Look up a MusicBrainz disc id or release id and output information.
 
 You can get the MusicBrainz disc id with whipper cd info.
 
-Example disc id: KnpGsLhvH.lPrNc1PBL21lb9Bg4-"""
+Example disc id: KnpGsLhvH.lPrNc1PBL21lb9Bg4-
+"""
 
     def add_arguments(self):
-        self.parser.add_argument(
-            'mbdiscid', action='store', help="MB disc id to look up"
-        )
+        self.parser.add_argument('-d', '--disc-id',
+                                 action='store', dest='disc_id',
+                                 help="MB disc id to look up", default=None)
+        self.parser.add_argument('-R', '--release-id',
+                                 action='store', dest='release_id',
+                                 help="MB release id to look up", default=None)
+
+    def _printMetadata(self, md):
```

Comment:
Maybe you can add a docstring here.

Reply:
I have added a docstring for this in my new commit too.

Reply:
Looks good, maybe use this (slight improvement):

```python
        """
        Print out received MusicBrainz's metadata in a sensible way.
        
        :param md: MusicBrainz's metadata about the disc
        :type md: `DiscMetadata`
        :rtype: None
        """
```

Reply:
\o/ okay i get it.

Reply:
Nitpick:
- I would use this as description in the docstring: `Print out received MusicBrainz's metadata in a sensible way.`
- Otherwise the first line of the docstring is missing a period (`.`) at the end (`sensible way.`)
- The `rtype` entry in the docstring is missing a semicolon (`:`) - should be `:rtype:`